releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,16 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 3.2.0 - February 29, 2024
+
+### Enhancements
+
+* Supported declaring feature flags with [Microsoft Feature Management schema](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureManagement.v1.0.0.schema.json)
+
+### Breaking Changes
+
+There are no breaking changes in this release.
```

Comment:
I can understand we made this statement in the previous release (4.0.0-preview), which is a major version update, but are we going to say this in every new release going forward? Feels like over kill.

Reply:
Removed.